server/routerlicious/packages/services/src/mongodb.ts

```diff
@@ -479,6 +541,9 @@ const DefaultKeepAliveInitialDelay = 60000;
 const DefaultSocketTimeoutMS = 100000;
 const DefaultConnectionTimeoutMS = 120000;
 const DefaultMinHeartbeatFrequencyMS = 10000;
+const DefaultApiCounterIntervalMS = 60000;
```

Comment:
If we can make this number larger? Maybe 10 minutes, since otherwise, it would be a lot of refreshment for different kinds of MongoDB methods based on current design, which should be costly.

Reply:
This is default and this value is configurable.

The reason why I use 1 minutes interval, is to:
1. Prevent loss metrics due to other reasons caused pod stop. There are not flush-like method for the collection so use a more frequent value benefits that
2. Detect the node issue faster. The reason is db layer is very busy, so even check every minutes, we should have a lot of numbers already. For the idle time, we have a protection of minimum failure count to consider